tests/accesscontrol/helper/helper.go

```diff
@@ -156,3 +181,247 @@ func DefineAndCreateServiceOnCluster(name, namespace string, port int32, targetP
 
 	return nil
 }
+
+// Finds the MachineConfigPools that manage worker nodes of the cluster. Returns a map of MCP names
+// to slices of worker names.
+func GetWorkersMCPs() (mcpNodes map[string][]string, err error) {
+	// Get the eco-goinfra client.
+	egiClient := globalhelper.GetEcoGoinfraClient()
+	if egiClient == nil {
+		return nil, fmt.Errorf("eco-goinrfa client is not initialized")
```

Comment:
`eco-goinrfa` --> `eco-goinfra`

Reply:
Thanks. Fixed in https://github.com/redhat-best-practices-for-k8s/certsuite-qe/pull/1059/commits/adee779dcdbe6f3b0536704cd7743230ebf8d04d